integrationtest/backend/elasticsearch/src/test/java/org/hibernate/search/integrationtest/backend/elasticsearch/management/ElasticsearchSchemaValidationIT.java

```diff
@@ -859,23 +853,6 @@ private void setupSimpleIndexWithLocalDateField(String indexName) {
 	}
 
 	private SearchSetupHelper.SetupContext withManagementStrategyConfiguration() {
-		return setupHelper.withDefaultConfiguration( BACKEND_NAME )
-				.withIndexDefaultsProperty(
-						BACKEND_NAME,
-						ElasticsearchIndexSettings.LIFECYCLE_STRATEGY,
-						ElasticsearchIndexLifecycleStrategyName.VALIDATE.getExternalRepresentation()
-				)
-				.withBackendProperty(
-						BACKEND_NAME,
-						// Don't contribute any analysis definitions, migration of those is tested in another test class
-						ElasticsearchBackendSettings.ANALYSIS_CONFIGURER,
-						new ElasticsearchAnalysisConfigurer() {
-							@Override
-							public void configure(ElasticsearchAnalysisDefinitionContainerContext context) {
-								// No-op
-							}
-						}
-				);
+		return setupHelper.withConfiguration( CONFIGURATION_ID );
```

Comment:
I really don't see why this commit was necessary?
It looks like you're duplicating a properties file for no reason, whereas just using the default one and overriding what's relevant to the test used to work just fine.

Reply:
I think that provide the configuration a `validate` schema strategy property type makes more readable the tests. But OK I'll revert this commit.